validations/semantic_resolved_usages_test.go

```diff
@@ -54,6 +54,66 @@ resources:
 		Ω(issues[1].Line).Should(Equal(28))
 	})
 
+	It("Sanity - resource of type configuration", func() {
+		mtaContent := []byte(`
```

Comment:
please simplify example and remove stuff not related to the scenario to be checked